app-editors/featherpad/featherpad-1.0.0.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xdg cmake
+
+DESCRIPTION="Lightweight Qt5 Plain-Text Editor for Linux"
+HOMEPAGE="https://github.com/tsujan/FeatherPad"
+SRC_URI="https://github.com/tsujan/FeatherPad/archive/V${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="app-text/hunspell
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtprintsupport:5
+	dev-qt/qtsvg:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtx11extras:5
+	x11-libs/libX11"
```

Comment:
Looking upstream [here](https://github.com/tsujan/FeatherPad/blob/master/CMakeLists.txt#L18) and [here](https://github.com/tsujan/FeatherPad/blob/master/featherpad/CMakeLists.txt#L26-L29) it looks like we can make those optional dependencies with an `IUSE="X"`.

Reply:
I'm aware of this - it is documented in the INSTALL file:

> *******************************
> *   Compilation without X11   *
> *******************************
> 
> If, for whatever reason, you do not want the X11 support, you could first turn on the option "WITHOUT_X11" with cmake:

I can add the USE flag. Will update the PR.